boefjes/pyproject.toml

```diff
@@ -63,6 +62,7 @@ netaddr = "0.9.0"
 defusedxml = "^0.7.1"
 pydantic-settings = "^2.0.3"
 opentelemetry-instrumentation = "^0.41b0"
+pillow = "^10.2.0"
```

Comment:
I think this is still only needed by the kat_webpage_analysis/check_images normalizer and should still be listed under that comment?

Reply:
You're right, I accidentally moved that comment line. It should be good now